modules/ROOT/pages/integration-testing.adoc

```diff
@@ -6,38 +6,36 @@
-MicroShed Testing helps to write and run integration tests for your applications, and streamlines your integration tests with the Testcontainers framework for an efficient workflow.
+MicroShed Testing helps to write and run true-to-production integration tests for your applications, and streamlines your integration tests with the Testcontainers framework for an efficient workflow.
 
 
 == Development and production parity
 
-link:https://12factor.net/dev-prod-parity[Development and production parity], is one of the factors in the link:https://12factor.net/[twelve-factor app], a methodology to build modern applications.
+link:https://12factor.net/dev-prod-parity[Development and production parity], is one of the factors in the link:https://12factor.net/[twelve-factor app], which is a methodology to build modern applications.
 The idea behind development and production parity is to keep the development, staging, and production environments similar, regarding time, personnel, and tools.
-To simplify the development process, developers often use tools in development different from production.
-For example, you might use a local Maven build in development, but the application might be deployed to a Docker container in production.
+To simplify the development process, developers often use tools in development that are different from production.
+For example, you might use a local Maven build to build a project for your application in development, but the application might be deployed to a link:https://www.docker.com/why-docker[Docker] container in production.
 Differences between the environments can cause a test to fail in production, though the test passes in the development environment.
-MicroShed Testing helps achieve development and production parity by testing the application in the environment similar to production.
+MicroShed Testing helps achieve development and production parity by testing the application in an environment similar to production.
 
-== Write integration tests with Microshed Testing
+== Write integration tests with the Microshed Testing library
 
 You can deploy your microservices in different environments with containers.
-With Testcontainers framework, you can use containers in a test environment.
+With the Testcontainers framework, you can use containers in a test environment.
 Microshed Testing is a Java library for MicroProfile and link:https://jakarta.ee/[Jakarta EE] developers to test their applications in an environment similar to production.
-Microshed Testing implements Testcontainers to support development and production parity in testing.
+Microshed Testing implements Testcontainers to support the development and production parity in testing.
 
-With MicroShed Testing framework, you can write an integration test that looks something like this:
+With MicroShed Testing, you can write an integration test that looks like the following:
```

Comment:
avoid use of `following` as a noun:

With MicroShed Testing, you can write an integration test that looks like the following example: